src/NuGet.Core/NuGet.Common/Migrations/Migration1.cs

```diff
@@ -156,13 +157,13 @@ private static void FixPermissions(string path, PosixPermissions umask)
             {
                 PosixPermissions correctedPermissions = permissions.Value.WithUmask(umask);
                 string correctedPermissionsString = correctedPermissions.ToString();
-                Exec("chmod", correctedPermissionsString + " " + path);
+                Exec("chmod", correctedPermissionsString + " " + "\"" + path + "\"");
             }
         }
 
         internal static PosixPermissions? GetPermissions(string path)
         {
-            string output = Exec("ls", "-ld " + path);
+            string output = Exec("ls", "-ld " + "\"" + path + "\"");
```

Comment:
This change hopefully addresses https://github.com/NuGet/NuGet.Client/commit/0966444b35d8a74fa25adf0368b983f389884377#r86528891 comment. I modified existing test to ensure path with spaces are working as expected.

Reply:
Same as above, I'd use a separate method we can test.

Reply:
I pushed a new commit adding a new extension method to perform this string formatting. Please review and let me know the feedback.